packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -687,6 +687,13 @@ const conf = (module.exports = convict({
       env: 'SUBSCRIPTIONS_MANAGEMENT_URL',
       format: String,
     },
+    allowUnauthenticatedRedirects: {
+      default: false,
+      doc:
+        'Whether to allow any redirects to Payments for an unauthenticated user',
+      env: 'SUBSCRIPTIONS_UNAUTHED_REDIRECTS',
+      formlat: Boolean,
+    },
```

Comment:
Added a feature flag because Payments currently requires authentication and a user who isn't signed in will be redirected to settings.  We don't want people end up in settings after sign up/in.